### DISCUSSION TITLE
ElasticSearch documents retention

### DISCUSSION THREAD

**Discussion by DashCarlos** [Security-Onion-Solutions/securityonion#2186] [Unsupported Versions] [answered]
Is there a way to limit the space that documents can occupy or limit the number of days they are stored? 
![imagen_2020-12-03_112314](https://user-images.githubusercontent.com/31094207/100996902-ef6a1380-3559-11eb-900a-526bec22550c.png)

**Comment by weslambert** [comment, marked as answer]
Hi @DashCarlos , 

This is handled through` log_size_limit`: https://docs.securityonion.net/en/latest/elasticsearch.html?highlight=log_size_limit#pillar-files

**Reply by TOoSmOotH** [reply]
@DashCarlos You can also limit the days per index in global.sls

**Reply by DashCarlos** [reply]
> Hi @DashCarlos ,
> 
> This is handled through` log_size_limit`: https://docs.securityonion.net/en/latest/elasticsearch.html?highlight=log_size_limit#pillar-files


is it a percentage value?

**Reply by weslambert** [reply]
It is a disk size (GB) value.
